Use correct GitHub casing (#6701)
umbraco/UmbracoDocs: 15/umbraco-deploy/installation/install-configure.md

**Set up the Git repository and Umbraco project**

> The first step to get Umbraco Deploy up and running is to set up a GitHub repository. This will be where source code is stored, and, following the GitHub Actions example, act as our environment where we will set up a CI/CD pipeline.

Reviewer: ```suggestion
The first step to get Umbraco Deploy up and running is to set up a GitHub repository. This will store the source code and serve as our environment for setting up a CI/CD pipeline, following the GitHub Actions example.
```
